src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+// booksSlice.js
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [
+  { id: '1', title: 'O Auto da Compadecida', author: 'Ariano Suassuna' },
+  { id: '2', title: 'Crime and Punishment', author: 'Fiodor Dostoyevsky' },
+  { id: '3', title: 'Mininos Sabidos', author: 'Judá Teixeira' },
+];
```

Comment:
- [ ] Kindly ensure your initial array is an empty array, this is a project requirement.